src/sparse/impl/KokkosSparse_spiluk_numeric_impl.hpp

```diff
@@ -382,8 +385,8 @@ void iluk_numeric ( IlukHandle& thandle,
 
   HandleDeviceEntriesType level_idx = thandle.get_level_idx();
 
-  typedef Kokkos::View< nnz_lno_t**, Kokkos::LayoutLeft, Kokkos::Device<execution_space,memory_space> > WorkViewType;
-    
+  typedef Kokkos::View< nnz_lno_t**, Kokkos::Device<execution_space,memory_space> > WorkViewType;
```

Comment:
Prefer C++11 - style `using` over `typedef`:
```c++
using WorkViewType = Kokkos::View<nnz_lno_t**, Kokkos::Device<execution_space,memory_space>>;
```
Note that C++11 permits the two trailing `>>` to be next to each other.  You don't need to leave a space between them.

Reply:
Thanks @mhoemmen . I replaced ```typdef```with ```using``` in the new commit.

Reply:
Thanks @vqd8a !